e2e/server.js

```diff
@@ -0,0 +1,29 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const cors = require('cors');
+
+const port = 8060;
+const app = express();
+
+const db = {
+  paymentAccounts: { data: { paymentAccounts: [] }, status: 200 }
+};
+
+app.use(bodyParser.json());
+app.use(cors());
+
+app.post('/api/setup/payment-accounts', (req, res) => {
```

Comment:
Make it generic `app.post('/stub'`.
Payload:
```
{ payload, status, url}
```

Reply:
Additionally I added `method` property to determine `GET`, `POST`...